.github/ISSUE_TEMPLATE/feature_request.yml

```diff
@@ -0,0 +1,58 @@
+name: 🌟 Feature request
+description: Suggest a feature to improve the app
+labels: [feature]
+body:
+  - type: textarea
+    id: context
+    attributes:
+      label: Context
+      description: (optional) If some context is required, describe it here
+      placeholder: |
+        Example:
+          "When I do X, I like to have Y, but the app has Z. This is also related to that time I went camping..."
+
+  - type: textarea
+    id: feature-description
+    attributes:
+      label: Describe your suggested feature
+      description: How can the app be improved?
+      placeholder: |
+        Example:
+          "It should work like this..."
+    validations:
+      required: true
+
+  - type: textarea
+    id: other-details
+    attributes:
+      label: Other details
+      description: "(optional)"
+      placeholder: Additional details and attachments.
+
+  - type: checkboxes
+    id: acknowledgements
+    attributes:
+      label: Acknowledgements
+      description: Read this carefully, we will close and ignore your issue if you skimmed through this.
+      options:
+        - label: >-
+            I have searched the existing issues and this is a new one, **NOT** a duplicate or related to another open issue.
+          required: true
+        - label: >-
+            This is a **single** feature request, 
+            in case of multiple feature requests I will open a separate issue for each one
+            (they can always link to each other if related)
+          required: true
+        # - label: >-
+        #     This is not a question or a discussion, in which case I should have gone to
+        #     [lemmy.ml/c/thumbkey](https://lemmy.ml/c/thumbkey)
+        #   required: true
```

Comment:
I enabled the discussion board now fwiw.

Reply:
Adressed by [873eea8](https://github.com/nightkr/flickboard/pull/39/commits/873eea8a1bae1bf8ae655c7b7c0323b6cba3a826)